src/v2/Apps/Artwork/routes.tsx

```diff
@@ -11,15 +11,25 @@ export const routes = [
       ArtworkApp.preload()
     },
     query: graphql`
-      query routes_ArtworkQuery($artworkID: String!) {
-        artwork(id: $artworkID) @principalField {
+      query routes_ArtworkQuery(
+        $artworkID: String!
+        $includeUnlisted: Boolean
+      ) {
+        artwork(id: $artworkID, includeUnlisted: $includeUnlisted)
+          @principalField {
           ...ArtworkApp_artwork
         }
         me {
           ...ArtworkApp_me
         }
       }
     `,
+    prepareVariables: ({ artworkID }, _props) => {
+      return {
+        artworkID,
+        includeUnlisted: true,
```

Comment:
@damassi I don't know what I'm doing here and would like to get your 👍 . 😅 

Basically in https://github.com/artsy/metaphysics/pull/2428, we introduced an optional `includeUnlisted` argument when querying an artwork. In Force, we wanna be able to show unlisted artworks on the artwork page (but render `noindex` meta tag for search engines). Is this the only place/query we fetch the artwork on the artwork page, or do I need to provide the `includeUnlisted` argument somewhere else too?